src/components/Calculator.js

```diff
@@ -1,39 +1,68 @@
-import React, { Component } from 'react';
+import { Component } from 'react';
+import calculate from '../logic/calculate';
+
 // eslint-disable-next-line react/prefer-stateless-function
 export default class Calculator extends Component {
+  constructor(props) {
```

Comment:
[OPTIONAL] You performed exceptionally. I am impressed with all the implementations you did. However, the use of eslint disabler on this project is not authorized. If you run into an issue where you could not find another solution, that allows for disabling eslint, please let me know so I can understand why it was necessary to do so. Thank you.